### PR TITLE
Fixing link

### PR DESCRIPTION
**Pull request by spekulatius**
Hey @ddeboer,

just fixing a link.

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,8 +32,7 @@ $bool = $validator->isValid('NL123456789B01');
 ```
 
 Additionally check whether the VAT number is in use, with a call to the [VAT
-Information Exchange System (VIES)]
-(http://ec.europa.eu/taxation_customs/vies/faq.html#item_16) SOAP web service:
+Information Exchange System (VIES)](http://ec.europa.eu/taxation_customs/vies/faq.html#item_16) SOAP web service:
```
Comment (ddeboer) [inline README.md:35]: They are identical. 😁 Did you mean `https://...` perhaps?

Comment (spekulatius) [reply README.md:35]: Yeah, it was the markdown for the link which was broken (not the actual URL) - the link was just automatically linked on the URL by github.

Comment (ddeboer) [reply README.md:35]: Ah, good catch!

**Comment by ddeboer** [comment]
Thanks!

**Comment by spekulatius** [comment]
No worries, happy to help out!

**Pull request merged**